### ISSUE TITLE
Re-organise into GitBook and Publish?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-amazon-web-services#3]
@nikhilaravi & @jackcarlisle we can use _this_ repo as our "General" AWS Learnings ...

**Comment by nelsonic** [comment]
A search for AWS on Gitbook.io yields a few results: https://www.gitbook.com/search?q=aws
but none of them have any traction or appear to be complete...
![gitbook-aws-21-results](https://cloud.githubusercontent.com/assets/194400/13300171/b6a2cb64-db36-11e5-8404-329ea12fc92f.png)
So there's a gap.

**Comment by nikhilaravi** [comment]
:+1:

**Comment by rjpcasalino** [comment]
Was this ever done? I just started using AWS a month or so ago, but started an internal Wiki at work that centralized lots of AWS docs. I like the idea of this GitBook combined with explaining AWS in plain english.

**Comment by nelsonic** [comment]
@Gusbenz _good question_ sadly, the answer is no. 😞 
not because we don't think it's a good idea, 💡 
rather it's because we got "distracted" by: https://github.com/dwyl/learn-aws-lambda ... 😉 
would _love_ some _help_ with it if _anyone_ has time.

**Comment by rjpcasalino** [comment]
Well, I'd like to get started if someone is willing to point me in the right direction 👉
